### PR TITLE
Feature/ui

### LINKED ISSUE #67: Build front-end UI
Elements
- [x] Headings
- [x] Paragraphs
- [x] Input Textfield (incl. focus)
- [x] Button
- [x] Sections (dark, light, grey)
- [x] Main
- [x] Container
- [x] List - stripped
- [x] Column (grid)
- [x] Icons

Components
- [x] Page Header
- [x] Section Header
- [x] Field
- [x] Creative Commons
- [x] Adlib
- [x] Circle Icon
- [x] Footer

### PR DESCRIPTION
**Pull request by dlouise64**
closes #67

# What has changed and why?

Added the following

Elements
- [x] Headings
- [x] Paragraphs
- [x] Input Textfield (incl. focus)
- [x] Button
- [x] Sections (dark, light, grey)
- [x] Main
- [x] Container
- [x] List - stripped
- [x] Column (grid)
- [x] Icons
- [x] Radio Group

Components
- [x] Page Header
- [x] Section Header
- [x] Field
- [x] Creative Commons
- [x] Adlib
- [x] Circle Icon
- [x] Footer

- [x] Tests

- [x] Madlib working
# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

**Comment by dlouise64** [comment]
@Br3nda @verbman going to add more test and also need to add radio and dropdowns. Will get madlib working on this PR too

**Comment by dlouise64** [comment]
@Br3nda @verbman I am moving this to review column now as there's another story to make it functional

https://waffle.io/ServiceInnovationLab/piccolo/cards/5bc555eb454d3f00a763a049

I've already started on that story but we are still deciding what to show in the results.

File: src/components/App.js [hunk 1]
```diff
@@ -1,16 +1,149 @@
...
+        return <Column key={i}>
+          <ColumnHeader
+            country={item.name}
+            eligible={item.eligible}
+            age={item.age}
+          />
+          {props.state.hasCitizenship && <Paragraph small>
+            If you are a citizen or permanent resident, not on ACC.
+          </Paragraph>}
+          {props.state.do_you_have_a_partner === 'yes' && <Paragraph small>
```
Comment (Br3nda) [inline src/components/App.js]: How aboht having consistent variable naming styles. How about `has_partner`

File: src/components/App.js [hunk 2]
```diff
@@ -1,16 +1,149 @@
...
+    />
+    <Grid>
+      {countries.map((item, i) => {
+        return <Column key={i}>
+          <ColumnHeader
+            country={item.name}
+            eligible={item.eligible}
+            age={item.age}
+          />
+          {props.state.hasCitizenship && <Paragraph small>
```
Comment (Br3nda) [inline src/components/App.js]: We were using snake case for the other variables but now this is camelCase

File: src/components/App.js (hunk 1 above)
Comment (Br3nda) [reply src/components/App.js]: Could it be normalised down to a boolean instead of looking for a string?

**Pull request merged**